cob_hardware_config/cob4-4/config/teleop.yaml

```diff
@@ -3,13 +3,15 @@ run_factor: 2.0
 
 # buttons
 deadman_button: 5
-run_button: 7
+#run_button: 7
 base_safety_button: 4
+recover_base_button: 7
```

Comment:
this changes the meaning of the buttons, right?
is this only special for cob4-4 or is this something to be used on the other robots, too?